repoman/flatpak.py

```diff
@@ -188,9 +191,14 @@ def __init__(self, parent, remote, name, option):
         content_grid.set_row_spacing(6)
         content_area.add(content_grid)
 
-        remote_title = flatpak.remotes.remotes[self.option][self.remote]['title']
-        description = flatpak.remotes.remotes[self.option][self.remote]['about']
-        url = flatpak.remotes.remotes[self.option][self.remote]['url']
+        remote_title = self.remote_data['title']
+        description = self.remote_data['about']
+        # FIXME: Pyflatpak has a bug, we want to work around this if present
+        # See https://github.com/pop-os/pyflatpak/issues/4
+        try:
+            url = self.remote_data['homepage']
```

Comment:
This kind of thing should really not make it into release. pyflatpak 0.4.1 will be out soon enough

Reply:
I feel a lot more comfortable having it there, at least until we can be reasonably certain people have upgraded to 0.4.1. Given we had the issue with 20.04 release and we can be reasonably certain several people are using the buggy version of pyflatpak, I think it would be better to avoid shipping software with a breakage

Reply:
We would release pyflatpak first so if they update to get this repoman they would also get the new pyflatpak...

Reply:
removed